src/ark/analysis/neighborhood_analysis.py

```diff
@@ -97,7 +97,8 @@ def create_neighborhood_matrix(all_data, dist_mat_dir, included_fovs=None, distl
 
     # Remove cells that have no neighbors within the distlim
     total_cell_count = cell_neighbor_counts.shape[0]
-    keep_cells = cell_neighbor_counts.drop([fov_col, cell_label_col], axis=1).sum(axis=1) != 0
+    keep_cells = cell_neighbor_counts.drop([fov_col, cell_label_col], axis=1).sum(
+        axis=1, numeric_only=True) != 0
```

Comment:
What non-numeric values can populate the `cell_neighbor_counts` dataframe?

Reply:
Oops, look like the cell cluster column wasn't being dropped and that's what was causing a weird warning message.